src/main/resources/standardTestCases/cont_while_possible/cont_while_possible_scatter.wdl

```diff
@@ -0,0 +1,24 @@
+task even_numbers_only {
+Int i
+  command {
+	  if [ $((${i}%2)) -eq 0 ]; then
+	    exit 1;
+	  fi
+	}
+	runtime {
+		docker: "ubuntu:latest"
+	}
+}
+
+workflow cont_while_possible_scatter {
+  Array[Int] myRange = range(4)
+
+	scatter(i in myRange){
+	  call even_numbers_only {input: i = i}
+	}
+
+	if (true) {
```

Comment:
@mcovarr  Originally I expected this code to touch logic involving conditionals as well--but it should totally be unaffected.